src/oc/change/api/change.clj

```diff
@@ -34,6 +43,13 @@
                                                :read read-data}}
                         false))
 
+  :delete! (fn [ctx] (let [user (:user ctx)
+                           user-id (:user-id user)
+                           delete-item (read/delete! post-uuid user-id)]
+                       (if (nil? delete-item)
+                         {:deleted-items :ok}
+                         {:deleted-items false})))
```

Comment:
@bago2k4 I'm not totally familiar with Liberator, but what is the purpose of modifying the context map here? It doesn't appear that `:deleted-items` is actually used anywhere.

Reply:
nope @cjsauer is not directly used, but it's part of the liberator process to add to the context the results of all the DB operations.
If you look in the other liberator resources you should find more like this.